eslint.config.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import pluginJs from '@eslint/js';
+import noOnlyTests from 'eslint-plugin-no-only-tests';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  {
+    ignores: [
+      '**/*.js',
+      '**/*.cjs',
+      '**/*.mjs',
+      '**/*.d.ts',
+      '**/.wireit/',
+      '**/node_modules/',
+      'lib/',
+      'vscode-extension/.vscode-test/',
+      'vscode-extension/lib/',
+      'vscode-extension/built/',
+    ],
+  },
+  {
+    files: ['**/*.ts'],
```

Comment:
should this be `src/**/*.ts`?

Reply:
I think `**/*.ts` is actually fine, because if we're globally ignoring all non-source files, then a general ".ts extension" filter seems perfectly fine.

This way we don't have to know anything about the project layout, which in this case would also require that we add `vscode-extension/src`.